phabricator-user.rst

```diff
@@ -48,7 +48,7 @@ at the top of any page.
    :alt: Screenshot of Phabricator login button
 
 You'll be taken to another page with a single button, which will in
-turn take you to `bugzilla.mozilla.org
+the turn takes you to `bugzilla.mozilla.org
```

Comment:
This isn't grammatically better than the original - "the" doesn't belong here.  Drop this change.

Reply:
Hi byron jones

"takes" is acceptable in place of "take"?

Reply:
No, take should be singular here (ie. there's no need to change this sentence).